examples/machine-learning/build-hcs-images/README.md

```diff
@@ -0,0 +1,63 @@
+# Build images for HCS (Hypercompute cluster service) cluster
+
+This document describes steps to deploy images for cluster deployed using
+Cluster Director (HCS). For each image it may take upto ~60 min to finish the
+build process and publish image in your project before you move ahead with
+deploying cluster.
+
+## Requirements
+
+Make sure `gcluster`, `gcloud`, `terraform` and `packer`
+binaries to be available in `PATH`.
+
+## Build A3M image
```

Comment:
Update this and following section:

1. Add sentence at top common to both commands saying "This process will build 2 images. One for a3-megagpu-8g and another one that can be used for all other machines. You may choose to run the two commands in parallel in 2 separate shells"
2. Just show the two commands
3.